README.md

```
@@ -163,11 +163,11 @@ This displays total episodes played, total listening time, starred episodes, sub
 
 ## Searching
 
-The `search` command performs full-text search across episodes, feeds, and chapters. The `save` and `extend` commands must be run prior to this.
+The `search` command performs full-text search across episodes, feeds, and chapters. Run `save` and `extend` first, and run `chapters` as well if you want chapter results.
 
     $ overcast-to-sqlite search "machine learning"
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)